### DISCUSSION TITLE
Sonoff TRV support

### DISCUSSION THREAD

**Discussion by paulzhdanov** [madchicken/homebridge-zigbee-nt#294] [Ideas]
Hi @madchicken , will it be hard to add Sonoff TRV (radiator thermostat head) support? I'm ready to be a tester, I have a device and will be glad to try.

**Comment by madchicken** [comment]
I need to look into the specification of the device (exposed endpoints) but it should be easy since the plugin already supports other thermostats. I will update this discussion as soon as I find some time to check!

**Reply by paulzhdanov** [reply]
Please let me know if I can do smth to help

**Reply by madchicken** [reply]
did you try adding it using the configurable device? You can find some info here https://github.com/madchicken/homebridge-zigbee-nt/wiki/Add-new-devices-in-configuration

**Reply by paulzhdanov** [reply]
No, but I can try. However it's not obvious enough for me... Did I got you right: 1) general config should be taken from thermostat [here](https://github.com/madchicken/homebridge-zigbee-nt/blob/master/src/accessories/configurable-accessory.ts), property list: https://www.zigbee2mqtt.io/devices/TRVZB.html from here. Correct?
